test/e2e_node/checkpoint_container.go

```diff
@@ -144,6 +211,12 @@ var _ = SIGDescribe("Checkpoint Container", nodefeature.CheckpointContainer, fun
 			// If the container engine has not implemented the Checkpoint CRI API
 			// we will get 500 and a message with
 			// '(rpc error: code = Unimplemented desc = unknown method CheckpointContainer'
+			// or
+			// '(rpc error: code = Unimplemented desc = method CheckpointContainer not implemented)'
+			// if the container engine returns that it explicitly has disabled support for it.
+			// or
+			// '(rpc error: code = Unknown desc = checkpoint/restore support not available)'
+			// if the container engine explicitly disabled the checkpoint/restore support
 			if (int(statusError.ErrStatus.Code)) == http.StatusInternalServerError {
```

Comment:
I am not sure how to handle this without skipping. As long as we test against CRI implementations that do not support that `CheckpointContainer` RPC, either because of not implementing or explicitly disabling it, we need a way to ignore the test result. Skipping seems the one thing which allows this.

Do you have another idea, how to distinguish between a failure and the not implemented state. That is the reasons I am checking for the error message. These error message are the known reasons if an engine has not implemented it or explicitly disables support. If another failure is returned it should appear as a test error.

Reply:
So its not clear to me how you configured these jobs..

Are all crio jobs able to run this?

I think I'd expect (if crio, these tests should work). Currently if containerd, they are not working. I'd explictly add that you expect this to fail for containerd.

That way you are making sure you know the expected behavior. skip is not great imo because as I said we will not report that anything went wrong.

Reply:
It definitely makes sense to think about this. Skipping is indeed not optimal as it might hide real errors. Let's find a better solution in a followup.

Reply:
cc: @mikebrow